packages/frontend/editor-ui/src/components/InputPanel.vue

```diff
@@ -157,7 +155,7 @@ const isMappingEnabled = computed(() => {
 	return true;
 });
 const isExecutingPrevious = computed(() => {
-	if (!workflowRunning.value) {
+	if (!workflowsStore.isWorkflowRunning) {
```

Comment:
these changes feel like they could be a separate PR, so land them as a separate commit in `master`.

same for all the other places where we move this flag from uiStore to workflowsStore.

Reply:
I know it feels like we could do it in a separate PR, but this flag has been part of the problem. It would take too long to reliably extract this. Would you be ok with leaving it as part of this PR?